lang/llvm-devel/Portfile

```diff
@@ -114,9 +113,6 @@ depends_build-append    port:python${py_ver_nodot} \
 set pythonfullpath      ${prefix}/bin/python${py_ver}
 configure.args-append   -DPython3_EXECUTABLE=${pythonfullpath}
 
-# https://trac.macports.org/ticket/65463
-use_xcode               yes
-
```

Comment:
I suggest you leave all this as Chis had it. I see no need to change this.

builds down to 10.11 as is (with new release of legacysupport)

Reply:
Re Xcode requirement, and doing without it: https://github.com/llvm/llvm-project/issues/57037

Reply:
thanks!